map/src/main/kotlin/edu/artic/map/util/ArticObjectMarkerGenerator.kt

```diff
@@ -0,0 +1,26 @@
+package edu.artic.map.util
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import de.hdodenhof.circleimageview.CircleImageView
+import edu.artic.map.R
+
+
+class ArticObjectMarkerGenerator(context: Context) : BaseMarkerGenerator(context){
+
+    init {
+        container = LayoutInflater.from(context)
+                .inflate(R.layout.marker_artic_object, null) as ViewGroup
+    }
+
+
+    fun makeIcon(imageViewBitmap: Bitmap): Bitmap {
+        container.findViewById<CircleImageView>(R.id.circularImage).setImageBitmap(imageViewBitmap)
```

Comment:
can we use kotlin extensions here? I think sooo

Reply:
I tried... and didn't allow me to use kotlinx here

Reply:
weird. ok. it should just work on any view also.

Reply:
I know i spent like 20 minutes trying to figure out why, but gave up and decided to get the feature done